FIX: Auswahl des Streamservers auf verfügbare Server beschränkt

GWRon opened the pull request:
- behebt Problem der "TimeOuts" die sich in der 2 minutigen "Arbeitet..."-Meldung niederschlug
- Ursache war, dass der Streamserver per Zufall ausgewaehlt worden ist. Die Zufallswerte enthielten ungueltige / nicht laenger erreichbare Adressen (1-25 und 34)

GWRon committed:
FIX: Auswahl des Streamservers auf verfügbare Server beschränkt

- behebt Problem der "TimeOuts" die sich in der 2 minutigen "Arbeitet..."-Meldung niederschlug
- Ursache war, dass der Streamserver per Zufall ausgewaehlt worden ist. Die Zufallswerte enthielten ungueltige / nicht laenger erreichbare Adressen (1-25 und 34)

Trekky12 commented:
much more stable when same Random Number is used for both variables:
https://github.com/Trekky12/plugin.video.rtl_now/commit/dd18c74e901b6c6a7848808aebf75fd736fc1b8c

GWRon committed:
Alle funktionierenden Server in ein Tupel gepackt

- beim Abspielen wird kurzzeitig eine Notification eingeblendet, die den ausgewaehlten Server anzeigt (hilfreich, wenn noch mehr wegbrechen)
- im letzten Commit waren ein paar Server enthalten, die nun nicht mehr verfuegbar sind (29-31)

GWRon commented:
Thanks Trekky12, I updated my code to reflect your suggestion to use the same server for both params. I also replaced the whole "random int"-part with a random selection out of a tupel containing the available servers. An additional notification will hopefully help to identify servers starting to break apart.
